code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -644,22 +647,109 @@
 /obj/item/clothing/suit/hooded/wintercoat/janitor
 	name = "janitors winter coat"
 	desc = "A purple-and-beige winter coat that smells of space cleaner."
-	icon_state = "coatjanitor"
-	inhand_icon_state = "coatjanitor"
+	icon_state = "wintercoat_jani"
+	inhand_icon_state = "wintercoat_jani"
 	allowed = list(/obj/item/toy, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/storage/fancy/cigarettes, /obj/item/lighter,/obj/item/grenade/chem_grenade,/obj/item/lightreplacer,/obj/item/flashlight,/obj/item/reagent_containers/glass/beaker,/obj/item/reagent_containers/glass/bottle,/obj/item/reagent_containers/spray,/obj/item/soap,/obj/item/holosign_creator,/obj/item/key/janitor,/obj/item/melee/flyswatter,/obj/item/paint/paint_remover,/obj/item/storage/bag/trash,/obj/item/reagent_containers/glass/bucket)
 	hoodtype = /obj/item/clothing/head/hooded/winterhood/janitor
 
 /obj/item/clothing/head/hooded/winterhood/janitor
 	desc = "A purple hood that smells of space cleaner."
-	icon_state = "winterhood_janitor"
+	icon_state = "winterhood_jani"
+
+/obj/item/clothing/suit/hooded/wintercoat/chef
+	name = "\improper chef's winter coat"
+	desc = "A white wintercoat that somehow isnt stained yet, maybe it will be one day."
+	icon_state = "wintercoat_chef"
+	inhand_icon_state = "wintercoat_chef"
+	allowed = list(/obj/item/reagent_containers/spray/plantbgone, /obj/item/plant_analyzer, /obj/item/seeds, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/cultivator, /obj/item/reagent_containers/spray/pestspray, /obj/item/hatchet, /obj/item/storage/bag/plants, /obj/item/graft, /obj/item/secateurs, /obj/item/geneshears)
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/chef
+
+/obj/item/clothing/head/hooded/winterhood/chef
+	desc = "A hood just as clean as the main body, smells of oil anyways."
+	icon_state = "winterhood_chef"
+
+/obj/item/clothing/suit/hooded/wintercoat/mime
+	name = "\improper mime's winter coat"
+	desc = "A black and white striped coat that the mimes made for themselves, who knows what this can do..."
+	icon_state = "wintercoat_mime"
+	inhand_icon_state = "wintercoat_mime"
+	armor = list("melee" = 20, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/mime
+
+/obj/item/clothing/head/hooded/winterhood/mime
+	desc = "A black and white hood that is slightly disorenting to look at, so you try not to look at it."
```

Comment:
```suggestion
	desc = "A black and white hood that is slightly disorienting to look at, so you try not to look at it."
```

Reply:
I'm gonna stop with the grammar edits here for now. You really ought to run this stuff through a spell-checker or something, though.